packages/jspsych/src/modules/extensions.ts

```diff
@@ -0,0 +1,32 @@
+export interface JsPsychExtensionParameters {
+  [key: string]: any;
+}
```

Comment:
There wouldn't be a need to export this then, am I right?
In this case, we could also do
```suggestion
type JsPsychExtensionParameters = Record<string, any>;
```